glustercli/cmd/volume.go

```diff
@@ -50,6 +51,7 @@ func init() {
 	// Volume Create
 	volumeCreateCmd.Flags().IntVarP(&flagCreateCmdStripeCount, "stripe", "", 0, "Stripe Count")
 	volumeCreateCmd.Flags().IntVarP(&flagCreateCmdReplicaCount, "replica", "", 0, "Replica Count")
+	volumeCreateCmd.Flags().IntVarP(&flagCreateCmdArbiterCount, "arbiter", "", 0, "Arbiter Count")
```

Comment:
Easier to understand if it's a bool.

Reply:
I think it would be better to retain this as an integer to maintain consistency with how it is used elsewhere in the gluster code (eg. in afr.c). Also, in volumeInfoDisplayNumbricks() we do display it as  a 'number' (i.e. the no. of arbiter bricks in a nx (2+1) volume).